lib/kennel/models/monitor.rb

```diff
@@ -27,11 +27,12 @@ class Monitor < Record
       }.freeze
       DEFAULT_ESCALATION_MESSAGE = ["", nil].freeze
       ALLOWED_PRIORITY_CLASSES = [NilClass, Integer].freeze
+      ALLOWED_UNLINKED = [] # rubocop:disable Style/MutableConstant placeholder for custom overrides
```

Comment:
If (when) we extend this validation to other linking types - e.g. monitor widgets in dashboards – what will ALLOWED_UNLINKED look like then?

Reply:
will need to move up into `Kennel::ALLOWED_UNLINKED` but the structure can stay the same